pkg/nftables/nftables.go

```diff
@@ -0,0 +1,204 @@
+package nftables
+
+import (
+	"errors"
+	"fmt"
+	"log/slog"
+	"net"
+
+	"github.com/google/nftables"
+	"github.com/google/nftables/expr"
+	"golang.org/x/sys/unix"
+)
+
+var Accept = nftables.ChainPolicyAccept
```

Comment:
nit: maybe make it a const (it is a const on original package anyway: https://github.com/google/nftables/blob/main/chain.go#L93C2-L93C19

Reply:
or maybe this can just be dropped, I can see you already use nftables.ChainPolicyAccept below